applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -899,12 +899,15 @@ void BaseSolidElement::CalculateOnIntegrationPoints(
                 double sigma_equivalent = 0.0;
 
                 if (dimension == 2) {
-                    sigma_equivalent = std::pow((stress_tensor(0,0) - stress_tensor(1,1)), 2.0) +
-                                                3*(stress_tensor(0,1) * stress_tensor(1,0));
+
+                    sigma_equivalent = std::pow(stress_tensor(0,0),2) 
```

Comment:
@loumalouomega @AlejandroCornejo, not for this PR, however please note that here a stress_tensor is allocated. this should be best avoided (allocation is by far the most expensive thing we do here) by using the voigt form which we have just above, without converting it into a tensor